### ISSUE TITLE
Official Dockerfile - test/production ready image v2 

### ISSUE THREAD

**Issue by lopo977** [openoereb/pyramid_oereb#1332]
We kindly ask to provide an official Dockerfile to build a production-ready image, as was done in pyramid_oereb v1.

Is this planned?
In this way, we can expose the web-service in a containerized environment and the user can test.

At the moment I'm testing only in my containerized development environment into my workstation.

Thanks again.
Luca

**Comment by lopo977** [comment]
Dear @voisardf ,
I talked to Mr. Tommasini to clarify where the Dockerfile for the pyramid_oereb V2 project is located.

I saw that the Dockerfile is in the same place of the 1st version, but it doesn't do the same thing.
https://github.com/openoereb/pyramid_oereb/blob/master/Dockerfile

The new Dockerfile v2 is designed for development, it installs an editor (vim) and a shell (zsh) in the image, it does not evaluate the templates (eval-templates) and it does not start the oereb-server.
e.g.` ENTRYPOINT ["eval-templates", "c2cwsgiutils_run"]`

I'm a little confused. The new /Dockerfile is intended to
`# keep container running until killed - For DEV use only`

for development purposes and stays in **sleep infinity**
`CMD ["sleep", "infinity"]`

The new Dockerfile is convenient for development (also to me), but is not intended to package the image, ready for deployment.

I'm sorry, but maybe I thought I'd find something else.
Or maybe I don't understand.
Thanks in advance for clarifications.
Kind regards,
Luca @lopo977
